openapi/ronin/types.bal

```diff
@@ -17,581 +17,581 @@
 # Estimates serve as proposals or quotes to be sent to Clients.
 public type Estimate record {
     # Tax
-    decimal tax?;
+    decimal? tax?;
```

Comment:
How this change happened without any changes in the openapi.yaml?
Did we manually change the types.bal? or used a new version of the openapi tool?